lib/events/dynamoevents/dynamoevents.go

```diff
@@ -93,6 +109,19 @@ func (cfg *Config) CheckAndSetDefaults() error {
 	if cfg.UIDGenerator == nil {
 		cfg.UIDGenerator = utils.NewRealUID()
 	}
+
+	// Table is required.
+	if cfg.Tablename == "" {
+		return trace.BadParameter("DynamoDB: table_name is not specified")
+	}
+
+	// If auto scaling is enabled, check all values are provided.
+	if cfg.EnableAutoScaling {
+		if cfg.ReadMaxCapacity == 0 || cfg.ReadMinCapacity == 0 || cfg.ReadTargetValue < 0.0 ||
```

Comment:
I'm not that familiar with dynamodb, but isn't 0 a valid value for read/write min capacity?

Reply:
Removed validation, leaving it to AWS to return an error when invalid values are passed.